### PR TITLE
refactor(pci.project.failover-ip): use country list given by API

### PR DESCRIPTION
**Pull request by marie-j**
Linked to MSB-90

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.html [hunk 1]
```diff
@@ -37,19 +37,11 @@
                 data-on-change="$ctrl.onRegionChange(modelValue)">
                 <span data-ng-bind="$item.name"></span>
             </oui-select>
-            <div class="oui-field__error" data-ng-class="{
-                'd-block': $ctrl.regionIsInvalid
-            }">
-                <span role="alert">
-                    <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
-                    <span data-translate="pci_projects_project_failoverip_order_legacy_region_invalid"></span>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.html]: Could be removed from `packages/manager/modules/pci/src/projects/project/failover-ips/translations/Messages_fr_FR.json` as well.

File: packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.controller.js [hunk 2]
```diff
@@ -25,28 +23,28 @@ export default class FailoverIpController {
...
 
   onInstanceChange(instance) {
-    if (this.ip.region) {
-      this.checkCompatibility(instance, this.ip.region);
-    }
-
-    const continent = find(this.regions, { name: instance.region }).continentCode;
-
-    this.REGIONS = REGIONS[continent].map(region => ({
+    const countries = find(this.regions, { name: instance.region }).ipCountries;
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.controller.js]: You can use destructuring assignment for instance:

```suggestion
    const { ipCountries: countries } = find(this.regions, { name: instance.region });
```

**Comment by antleblanc** [comment]
Is there any internal ticket reference related to this pull request?

**Comment by marie-j** [comment]
> Is there any internal ticket reference related to this pull request?

@antleblanc I updated the description

**Pull request merged**